src/main/java/bookstore/controller/BookController.java

```diff
@@ -19,42 +22,49 @@
 
 @RestController
 @RequestMapping("/api/books")
+@Tag(name = "Book Controller", description = "Endpoints for managing books")
 public class BookController {
     private final BookService bookService;
 
     public BookController(BookService bookService) {
         this.bookService = bookService;
     }
 
+    @Operation(summary = "Get all books")
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    public List<BookDto> getAll(Pageable pageable) {
+        return bookService.findAll(pageable);
     }
 
+    @Operation(summary = "Get a book by ID")
     @GetMapping("/{id}")
     public BookDto getById(@PathVariable Long id) {
         return bookService.get(id);
     }
 
+    @Operation(summary = "Create a new book")
```

Comment:
swagger has much more options to describe your API. 

For example, you can add in which case, which code or result will be returned back.
It makes your REST docs much more useful.

Reply:
it will be helpful in case of non-idempotent methods